python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint_search.py

```diff
@@ -997,71 +1189,137 @@ def getDistance(idx1, idx2, u):
             )[2][0]
             return distance
 
-        def getAngle(idx1, idx2, idx3, u):
-            """Angle in radian."""
+        def _getAngle(idx1, idx2, idx3, u):
+            """Angle in radians."""
             C = u.atoms[idx1].position
             B = u.atoms[idx2].position
             A = u.atoms[idx3].position
-            BA = A - B
-            BC = C - B
-            angle = _np.arccos(_np.dot(BA, BC) / (_norm(BA) * _norm(BC)))
+            angle = _calc_angles(C, B, A, box=u.dimensions)
```

Comment:
I think you could just do `sum(u.atoms[idx] for idx in [idx1, idx2, idx3]).angle.value()`